src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(operation, x) {
+      operation(x);
+
+      return this;
+    },
+
+    add(x) {
+      calculator.result += x;
+
+      return this;
+    },
+    subtract(x) {
+      calculator.result -= x;
+
+      return this;
```

Comment:
There is no need to return this everywhere. Only operate and reset methods should do it
![image](https://github.com/mate-academy/js_advanced_calculator/assets/96409563/21b01eed-a10e-456f-be51-d1244366da92)